static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -11,9 +11,11 @@ import { connatixTest } from 'common/modules/experiments/tests/connatix-ab-test'
 import { remoteEpicVariants } from 'common/modules/experiments/tests/remote-epic-variants';
 import { signInGate } from 'common/modules/experiments/tests/sign-in-gate';
 import { contributionsCovidBannerRoundTwo } from 'common/modules/experiments/tests/contribs-banner-covid-round-two';
+import { commercialRedplanet } from 'common/modules/experiments/tests/commercial-redplaner-aus';
```

Comment:
```suggestion
import { commercialRedplanet } from 'common/modules/experiments/tests/commercial-redplanet-aus';
```
Needs filename changing also